### PR TITLE
[Merged by Bors] - chore: fix many porting notes `was decide`

### PR DESCRIPTION
**Pull request by grunweg**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by grunweg** [comment]
Sorry, something went wrong with my git here. @kim-em Please feel free to cherry-pick my commit onto the right `bump` branch.

**Comment by jcommelin** [comment]
You only need to set the base branch of your PR. Which you can do by editing the title of you PR on github. I just did it for you.

**Comment by grunweg** [comment]
Thanks; I wasn't aware I could change the base branch that way. TIL!

File: Counterexamples/HomogeneousPrimeNotPrime.lean [hunk 1]
```diff
@@ -110,13 +110,7 @@ theorem grading.right_inv : Function.RightInverse (coeLinearMap (grading R)) gra
...
   · simp only [map_zero]
   · rcases i with (_ | ⟨⟨⟩⟩) <;> rcases zz with ⟨⟨a, b⟩, hab : _ = _⟩ <;> dsimp at hab <;>
-      cases hab <;>
-      -- Porting note: proof was `decide`
-      -- now we need a `simp` and two `erw` subproofs...
-      simp only [coeLinearMap_of, decompose, AddMonoidHom.coe_mk,
-        ZeroHom.coe_mk, sub_self, sub_zero]
-    · erw [map_zero (of (grading R ·) 1), add_zero]; rfl
-    · erw [map_zero (of (grading R ·) 0), zero_add]; rfl
+      cases hab <;> decide +revert
```
Comment (b-mehta) [inline Counterexamples/HomogeneousPrimeNotPrime.lean:113]: In this case it looks to me like the porting note should stay: the proof using `decide` did not do any reverts but the new proof `decide +revert` does. So this is still an example of the `decide` tactic being weaker than it used to be.

**Comment by grunweg** [comment]
The build on this PR fails, but for errors that looks very unrelated (and should be pre-existing).

**Comment by b-mehta** [comment]
I'm happy with this now, although I also don't know how to fix the bumping stuff. Maybe Johan or Kim can help?

**Comment by kim-em** [comment]
I've just merged `nightly-testing` back into this branch.

**Pull request closed without merging**